### PR TITLE
UI: add ru-RU locale for Control UI

### PR DESCRIPTION
**Pull request by vitalcc55**
AI-assisted PR.

## Summary

Describe the problem and fix in 2–5 bullets:

- Problem: Control UI does not offer a Russian locale even though the i18n system already supports lazy-loaded locale packs.
- Why it matters: Russian-speaking users cannot use a localized dashboard for the strings that already flow through the existing i18n layer.
- What changed: added a new `ru-RU` locale pack, registered it in the locale registry, added browser auto-detect for `ru*`, added language labels in existing locale files, and added targeted i18n test coverage.
- What did NOT change (scope boundary): this PR does not localize hardcoded English strings in views that are still outside the existing i18n key path.

## Change Type (select all)

- [ ] Bug fix
- [x] Feature
- [ ] Refactor
- [ ] Docs
- [ ] Security hardening
- [ ] Chore/infra

## Scope (select all touched areas)

- [ ] Gateway / orchestration
- [ ] Skills / tool execution
- [ ] Auth / tokens
- [ ] Memory / storage
- [ ] Integrations
- [ ] API / contracts
- [x] UI / DX
- [ ] CI/CD / infra

## Linked Issue/PR

- Closes #: None
- Related #: None

## User-visible / Behavior Changes

- Adds a `Russian (ru-RU)` option to the Control UI locale picker.
- Auto-detects `navigator.language` values starting with `ru` and resolves them to `ru-RU`.
- Ships a Russian locale bundle for already-internationalized Control UI strings.

## Security Impact (required)

- New permissions/capabilities? (`Yes/No`): No
- Secrets/tokens handling changed? (`Yes/No`): No
- New/changed network calls? (`Yes/No`): No
- Command/tool execution surface changed? (`Yes/No`): No
- Data access scope changed? (`Yes/No`): No
- If any `Yes`, explain risk + mitigation:

### Environment

- OS: Windows 11
- Runtime/container: local `pnpm` workspace checkout
- Model/provider: n/a
- Integration/channel (if any): Control UI / dashboard
- Relevant config (redacted): none

### Steps

1. Add `ru-RU` to the Control UI locale registry and locale type.
2. Add the new `ui/src/i18n/locales/ru-RU.ts` locale pack and locale label entries in existing locale files.
3. Build the UI and verify that the Russian locale chunk is emitted.

### Expected

- `ru-RU` is a supported locale.
- Russian browser locales resolve to `ru-RU`.
- UI build emits a lazy-loaded Russian locale asset.

### Actual

- `ru-RU` is registered and included in the locale picker labels.
- `pnpm ui:build` emits `dist/control-ui/assets/ru-RU-BULB5YzY.js`.
- `pnpm check` passes.

## Evidence

Attach at least one:

- [x] Failing test/log before + passing after
- [ ] Trace/log snippets
- [ ] Screenshot/recording
- [ ] Perf numbers (if relevant)

Evidence notes:

- `pnpm ui:build`
- `pnpm check`

## Human Verification (required)

What you personally verified (not just CI), and how:

- Verified scenarios:
  - Ran `pnpm ui:build`.
  - Ran `pnpm check`.
  - Confirmed the build emits `dist/control-ui/assets/ru-RU-BULB5YzY.js`.
- Edge cases checked:
  - `resolveNavigatorLocale("ru-RU")`
  - `resolveNavigatorLocale("ru")`
  - saved `ru-RU` locale coverage added in `ui/src/i18n/test/translate.test.ts`
- What you did **not** verify:
  - I did not do a full manual sweep of every Control UI view.
  - Some views still contain hardcoded English strings outside the current i18n coverage, so those remain untranslated.
  - I could not complete `codex review --base origin/main` locally because `codex.exe` failed to start in this environment with an access-denied error.

## Review Conversations

- [x] I replied to or resolved every bot review conversation I addressed in this PR.
- [x] I left unresolved only the conversations that still need reviewer or maintainer judgment.

If a bot review conversation is addressed by this PR, resolve that conversation yourself. Do not leave bot review conversation cleanup for maintainers.

## Compatibility / Migration

- Backward compatible? (`Yes/No`): Yes
- Config/env changes? (`Yes/No`): No
- Migration needed? (`Yes/No`): No
- If yes, exact upgrade steps:

## Failure Recovery (if this breaks)

- How to disable/revert this change quickly: revert this PR or remove `ru-RU` from the locale registry and locale type.
- Files/config to restore: `ui/src/i18n/lib/types.ts`, `ui/src/i18n/lib/registry.ts`, `ui/src/i18n/locales/ru-RU.ts`, locale label entries, and `ui/src/i18n/test/translate.test.ts`.
- Known bad symptoms reviewers should watch for: locale picker shows `ru-RU` but strings fall back to English because the locale chunk did not load.

## Risks and Mitigations

List only real risks for this PR. Add/remove entries as needed. If none, write `None`.

- Risk: Not all Control UI strings are currently wired through i18n, so some screens will still show English text.
  - Mitigation: This PR is intentionally scoped to the existing i18n surface only and does not claim full UI translation coverage.

### CODE REVIEW

File: ui/src/i18n/locales/ru-RU.ts [hunk 1]
```diff
@@ -0,0 +1,346 @@
+import type { TranslationMap } from "../lib/types.ts";
+
+export const ru_RU: TranslationMap = {
+  common: {
+    version: "Версия",
+    health: "Состояние",
+    ok: "OK",
+    offline: "Не в сети",
+    connect: "Подключиться",
```
[the remark this answers is not in the document]
Comment (vitalcc55) [reply ui/src/i18n/locales/ru-RU.ts:10]: Confirmed. `renderSidebarConnectionStatus` reads `t("common.online")`, and the Russian locale indeed did not define that key on the current PR head. I synced `ru-RU` with the current `en.ts` dashboard/status keyset in [80bc8e7a5c](https://github.com/openclaw/openclaw/commit/80bc8e7a5c688a7d71f8c57ca385a959ea61aec1), which adds `common.online` and the other recently added user-facing dashboard labels.

**Comment by vincentkoc** [comment]
Thanks for your contribution, however ee are extending language locales at our descretion and at this time will be closing this pull request.
